src/components/Login/login.css

```diff
@@ -0,0 +1,51 @@
+@import url("/src/styles/main.css");
```

Comment:
Would suggest not importing any css other files in existing css. It'll show clashes while overriding. (you can see it in elements).